taxi/urls.py

```diff
@@ -53,13 +53,25 @@
     ),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
     path(
-        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
+        "drivers/<int:pk>/",
+        DriverDetailView.as_view(),
+        name="driver-detail"
+    ),
+    path(
+        "drivers/",
+        DriverListView.as_view(),
+        name="driver-list"
+    ),
```

Comment:
There is a duplicate URL pattern for the driver list view. Ensure that each URL pattern is unique to avoid conflicts.